manticore/models/sql/compiler.py

```diff
@@ -90,3 +94,17 @@ def __maybe_set_limits(self, with_limits):
     def _compile_in(self, node: lookups.In):
         lookup = InFunction(node.lhs, node.rhs)
         return lookup.as_sql(self, self.connection)
+
+    def _compile_dict(self, param_dict: dict):
+        self._check_field_models(param_dict.keys())
+        params_str = ', '.join([f'`{k}`=%s' for k in param_dict.keys()])
```

Comment:
1. There is special function `quote_name` that adds back ticks.
2. `keys()` call is not necessary for iterating a dict.

Reply:
Fixed

Reply:
`keys()` still here

Reply:
deleted in https://github.com/just-work/django-manticore/pull/20/commits/4d5ad1c9ee88c4af2bf19c11b77a69ce6d26d600#diff-db7a7bdd6aae8cd6bf916bfbcd56bb58a6de152c51d5aa641086e277f1cee1b7R100

Reply:
```
    def _compile_dict(self, param_dict: dict):
        self._check_field_models(param_dict.keys())
```

Reply:
fixed